platform/commonUI/general/src/controllers/ObjectInspectorController.js

```diff
@@ -108,8 +110,13 @@ define(
 
                 getMetadata();
             });
-        }
 
+            this.InspectorMutation.listen(function () {
```

Comment:
`listen` returns a function to stop listening. This should get called from a `$destroy` listener to avoid accumulating stale listeners